zebra/zebra_gre.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * Zebra GRE
+ * Copyright (C) 2020 6WIND
+ *
+ * This file is part of FRR.
+ *
+ * FRR is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2, or (at your option) any
+ * later version.
+ *
+ * FRR is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with FRR; see the file COPYING.  If not, write to the Free
+ * Software Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA
+ * 02111-1307, USA.
+ */
+
+#include <zebra.h>
+#include <if.h>
+
+#include "zebra/debug.h"
+#include <zebra/zebra_gre.h>
+
+
+/* definitions */
+
+/*
+ * Handle GRE interface update - change to tunnel IP
+ */
+int zebra_gre_if_update(struct interface *ifp, uint16_t chgflags)
```

Comment:
so ... this is sort of shell of a file, and there's no explanation of its purpose. and the PR description is ... empty, so there's no information there about what is going on either. what do you hope a reviewer will do?

Reply:
let suppress this file for now.
as it is today, there is no added value in keeping it.